docker/ubuntu/bionic/Dockerfile

```diff
@@ -0,0 +1,132 @@
+# Docker build environment for Boost on Ubuntu Bionic
+#
+# Copyright (c) 2018 James E. King III
+#
+# Use, modification, and distribution are subject to the
+# Boost Software License, Version 1.0. (See accompanying file
+# LICENSE_1_0.txt or copy at http://www.boost.org/LICENSE_1_0.txt)
+
+FROM ubuntu:bionic
+MAINTAINER James E. King III <jking@apache.org>
+ENV CONTAINER_USER=boost
+ENV DEBIAN_FRONTEND noninteractive
+WORKDIR /boost
+
+# Make sure we're current
+
+RUN apt-get update && \
+    apt-get dist-upgrade -y && \
+    apt-get install -y --no-install-recommends \
+      apt-utils
+
+# Basic utilities
+
+RUN apt-get install -y --no-install-recommends \
```

Comment:
Instead of multiple RUN commands for each stanza, can we just have the continuation line continue? This will result in less filesystem unions I believe?

Reply:
I'm not sure that's necessary... given the architecture those unions should be highly optimized by now.  For example in the Apache Thrift docker files I think there is one statement per language at a minimum so there have to be at least 35 directives in the Dockerfile, and I don't notice any degradation when I am running.  (Of course, I'm usually running on flash...)